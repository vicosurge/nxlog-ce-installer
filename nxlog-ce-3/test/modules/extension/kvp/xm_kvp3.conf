include tmp/common.conf
<Extension kvp>
    Module	xm_kvp
    KVDelimiter =
    KVPDelimiter ;
    EscapeChar \\
    ValueQuoteChar "
</Extension>

<Extension json>
    Module	xm_json
</Extension>

<Input in>
    Module	im_file
    File	"modules/extension/kvp/xm_kvp3.in"
    SavePos	FALSE
    ReadFromLast FALSE
    Exec	if $raw_event =~ /^#/ drop(); \
                else \
                { \
                   kvp->parse_kvp(); delete($EventReceivedTime); delete($SourceModuleName); delete($SourceModuleType); kvp->to_kvp(); \
                }
</Input>

<Output out>
    Module	om_file
    File	'tmp/output'
</Output>

<Route 1>
    Path	in => out
</Route>