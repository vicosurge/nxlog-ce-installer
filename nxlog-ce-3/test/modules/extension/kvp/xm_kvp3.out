key1=value1;key2=value2;
key1="value 1";key2="value 2";key3="value 3";
key1=value\"1;key2=\"value2\";
